Mathlib/LinearAlgebra/Basic.lean

```diff
@@ -897,15 +897,8 @@ variable {re₁₂ : RingHomInvPair σ₁₂ σ₂₁} {re₂₁ : RingHomInvPai
 variable {re₃₄ : RingHomInvPair σ₃₄ σ₄₃} {re₄₃ : RingHomInvPair σ₄₃ σ₃₄}
 variable (e e₁ : M ≃ₛₗ[σ₁₂] M₂) (e₂ : M₃ ≃ₛₗ[σ₃₄] M₄)
 
--- @[simp] -- Porting note (#10618): simp can prove this
-theorem map_neg (a : M) : e (-a) = -e a :=
-  e.toLinearMap.map_neg a
-#align linear_equiv.map_neg LinearEquiv.map_neg
-
--- @[simp] -- Porting note (#10618): simp can prove this
-theorem map_sub (a b : M) : e (a - b) = e a - e b :=
-  e.toLinearMap.map_sub a b
-#align linear_equiv.map_sub LinearEquiv.map_sub
+#align linear_equiv.map_neg map_negₓ
+#align linear_equiv.map_sub map_subₓ
```

Comment:
What's the significance of the little `x` here?

Reply:
The porting wiki contains this:
"when mathport suggests that you add ₓ in the name of an #align statement, figure out why it wants that and leave a porting note. Common reasons are:
- a change in the order of universe variables, typeclass arguments or implicit arguments
- some other defeq change, due to a change deep down in core or std."

Reply:
It means that the types don't match up exactly; in this case the new lemma is much more general.